detectron2/data/detection_utils.py

```diff
@@ -579,8 +579,13 @@ def build_augmentation(cfg, is_train):
         max_size = cfg.INPUT.MAX_SIZE_TEST
         sample_style = "choice"
     augmentation = [T.ResizeShortestEdge(min_size, max_size, sample_style)]
-    if is_train:
-        augmentation.append(T.RandomFlip())
+    if is_train and "none" not in cfg.INPUT.RANDOM_FLIP:
```

Comment:
```suggestion
    if is_train and cfg.INPUT.RANDOM_FLIP != "none":
```

Reply:
Oops, that was left by accident from the previous commit. Should be correct now. :)